packages/fetchai/skills/gym/skill.yaml

```diff
@@ -8,13 +8,9 @@ behaviours: {}
 handlers:
   gym:
     class_name: GymHandler
-    args:
-      foo: bar
-tasks:
-  gym:
-    class_name: GymTask
```

Comment:
since the `Task` class is not loaded, there's no need to have this part in the configuration file.